crates/normalize-path/Cargo.toml

```
@@ -0,0 +1,10 @@
+[package]
+name = "normalize-path"
+description = "Like canonicalize, but without performing I/O"
+repository = "https://github.com/cargo-bins/cargo-binstall"
+documentation = "https://docs.rs/normalize-path"
+version = "1.0.0"
```

Same here:

```suggestion
version = "0.1.0"
```

The name `normalize-path` is available.